### PR TITLE
RawSegment definition and binary spec

### PR DESCRIPTION
**Pull request by wemrysi**
[ch7174]

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -18,3 +18,28 @@ lazy val core = project
...
+  .in(file("testkit"))
+  .dependsOn(core)
+  .settings(
+    performMavenCentralSync := false,
+    publishAsOSSProject := true)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.scodec" %% "scodec-bits" % "1.1.12",
+      "org.specs2" %% "specs2-core" % "4.3.6",
+      "org.specs2" %% "specs2-scalacheck" % "4.3.6"))
```
Comment (alissapajer) [inline build.sbt]: Could you stick the specs2 version in a val? Easier to upgrade that way.

File: testkit/src/main/scala/suprem/RawSegmentDecodeSpec.scala [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+import scala.util.{Left, Right}
+
+import java.lang.SuppressWarnings
+
+import org.specs2.mutable.Specification
+
+import scodec.bits._
+
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+trait RawSegmentDecodeSpec extends Specification {
```
Comment (alissapajer) [inline testkit/src/main/scala/suprem/RawSegmentDecodeSpec.scala]: Happy to have this all structured as a testkit right off the bat!

**Comment by alissapajer** [comment]
@wemrysi maybe try closing this and reopening it as a new PR to see if that triggers a travis build? Or opening a new whitespace PR to see if that triggers a build.

**Comment by wemrysi** [comment]
Closing, will reopen to hopefully trigger a build.

**Pull request closed without merging**